### PR TITLE
Fix add role permissions chips filtering.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-10493

### Changes
- use checkbox filter instead of group filter (they are the same as far as I can tell and the checkbox actually works)

add-role-permi

### CODE REVIEW

**Comment by john-dupuy** [comment]
Nice @Hyperkid123! Can we also do the same for the filter on the role details page?

**Comment by Hyperkid123** [comment]
Ah I see that is broken as well. Sure thing.

**Comment by Hyperkid123** [comment]
I see this is almost copy/paste. I think it would deserve refactoring these components (different task).

**Review by john-dupuy — APPROVED** [review]
Thanks @Hyperkid123 :rocket:

File: sitemap.xml [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-<url> <loc>https://cloud.redhat.com/roles/edit/:id</loc> </url>
-<url> <loc>https://cloud.redhat.com/roles/detail/:uuid</loc> </url>
-<url> <loc>https://cloud.redhat.com/roles/detail/:id/remove</loc> </url>
-<url> <loc>https://cloud.redhat.com/roles/detail/:id/edit</loc> </url>
-<url> <loc>https://cloud.redhat.com/roles/detail/:roleId/permission/:permissionId</loc> </url>
-<url> <loc>https://cloud.redhat.com/roles/detail/:roleId/permission/:permissionId/edit</loc> </url>
-<url> <loc>https://cloud.redhat.com/users</loc> </url>
-<url> <loc>https://cloud.redhat.com/users/detail/:username</loc> </url>
-<url> <loc>https://cloud.redhat.com/my-user-access</loc> </url>
+<url> <loc>https://cloud.redhat.com/undefined/undefined/</loc> </url>
```
Comment (karelhala) [inline sitemap.xml:3]: No need to push this file.

**Review by karelhala — COMMENTED** [review]
Looking good! The Group filter was probably used because previously there was `show more` option which is no longer in use.

File: sitemap.xml (hunk 1 above)
Comment (Hyperkid123) [reply sitemap.xml:3]: Oh yeah... It's generated when you run install.

**Pull request merged**